addons/subscription/data/subscription_demo.xml

```diff
@@ -1,23 +1,21 @@
 <?xml version="1.0" ?>
-<openerp>
-    <data>
+<odoo>
         <record id="base.user_demo" model="res.users">
```

Comment:
This adds the demo user to the group "group_sale_salesman" and the group "group_sale_salesman" has been assigned special permissions in the "security csv file".

This was done in task: 

https://www.odoo.com/web?#id=1771&view_type=form&model=project.task&action=327&menu_id=4720

And commit:  

https://github.com/odoo-dev/odoo/commit/eeaa68471d8d36ae37335d78fc0ccc128aec4dcd